plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java

```diff
@@ -543,6 +559,20 @@ public String getEffectivePom(String projectPath) {
     return result;
   }
 
+  private void reComputeDiagnostics(String pomPath) {
+    String pomUri = prefixURI(pomPath);
+    Type type = new TypeToken<PublishDiagnosticsParams>() {}.getType();
+    PublishDiagnosticsParams diagnostics =
+        doGetOne(Commands.RECOMPUTE_POM_DIAGNOSTICS, singletonList(pomUri), type);
+    Optional<InitializedLanguageServer> lServer = findInitializedLanguageServer();
+    if (!lServer.isPresent()) {
+      LOG.error("Language server not initialized.");
+      return;
+    }
+    String serverId = lServer.get().getId();
```

Comment:
`lServer` looks strange to me. Why not `langServer` or `languageServer` or even `ls`?

Reply:
fixed